### PR TITLE
Added new layouts and edited some of the MainActivity functionality.

### PR DESCRIPTION
**Pull request by teamagam3**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/58" target="_blank">ReviewNinja

File: app/src/main/res/layout/activity_main_content.xml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+            <android.support.v7.widget.Toolbar
+                android:id="@+id/main.toolbar"
+                android:layout_width="match_parent"
+                android:layout_height="?attr/actionBarSize"
+                app:layout_collapseMode="pin"
+                app:popupTheme="@style/ThemeOverlay.AppCompat.Light" />
+        </android.support.design.widget.AppBarLayout>
+
+        <RelativeLayout
```
Comment (teamagam1) [inline app/src/main/res/layout/activity_main_content.xml]: you should avoid using Relative layout.
why not linear layout?

Comment (teamagam1) [inline app/src/main/res/layout/activity_main_content.xml]: !fix

File: app/src/main/res/layout/activity_main_content.xml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="bottom"
+    sothree:umanoAnchorPoint="0.5"
+    sothree:umanoDragView="@+id/message_title_layout"
+    sothree:umanoFadeColor="@android:color/transparent"
+    sothree:umanoPanelHeight="68dp"
+    sothree:umanoShadowHeight="8dp">
+
+    <android.support.design.widget.CoordinatorLayout
```
Comment (teamagam1) [inline app/src/main/res/layout/activity_main_content.xml:16]: why both CoordinatorLayout  and inner layout?

File: app/src/main/res/layout/activity_main_content.xml [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+        </RelativeLayout>
+    </android.support.design.widget.CoordinatorLayout>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <android.support.design.widget.CoordinatorLayout
```
Comment (teamagam1) [inline app/src/main/res/layout/activity_main_content.xml]: you dont need CoordinatorLayout inside linear layout. it should handle it by itself. 

if it takes too much time. dont worry.

Comment (teamagam3) [inline app/src/main/res/layout/activity_main_content.xml]: AppBar might be added in the future, if we want to change the panel to be like activity

File: app/src/main/res/layout/activity_main_content.xml (hunk 1 above)
Comment (teamagam1) [inline app/src/main/res/layout/activity_main_content.xml]: !completed

**Pull request merged**